### DISCUSSION TITLE
problem with primereact menubar

### DISCUSSION THREAD

**Discussion by triguimed** [primefaces/community#2388] [PrimeReact] [answered]
when refreshing page the menubar is displayed as mobile version then it is displayed as desktop version

![image](https://github.com/user-attachments/assets/30e22f3b-5f4c-4d87-821c-71d6a0fe2af7)

**Comment by melloware** [comment]
Do you have a stackblitz reproducer?

**Reply by triguimed** [reply]
https://stackblitz.com/edit/ggkg4h?file=src%2FApp.jsx
it's the same code of primereact https://primereact.org/menubar/
Do multiple refresh and you will see that

**Reply by melloware** [reply]
OK then open a GitHub ticket please.

**Reply by melloware** [reply, marked as answer]
https://github.com/primefaces/primereact/issues/6956
